<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductItem</class>
 <widget class="QWidget" name="ProductItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>250</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #808080;</string>
  </property>
  <widget class="QLabel" name="product_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px;</string>
   </property>
   <property name="text">
    <string>Product Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="product_desc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QPushButton" name="product_add_to_cart_btn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add to cart</string>
   </property>
  </widget>
  <widget class="QPushButton" name="product_cancel_btn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 18px</string>
   </property>
   <property name="text">
    <string>â‚´</string>
   </property>
  </widget>
  <widget class="QLabel" name="product_price">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 18px</string>
   </property>
   <property name="text">
    <string>Price</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
